explainer.md

```diff
@@ -217,13 +187,16 @@ mediaSource.onsourceopen = (e) => {
   const sourceBuffer = mediaSource.addSourceBuffer(mime);
   
   const transport = new Http3Transport("/video");
-  while (transport.state == "connecting" || transport.state == "connected") {
-    const datagrams = await transport.receiveDatagrams();
-    for (let data of datagrams) {
-      if (data) {
-        const chunk = ccontainerizeMedia(data);
-        sourceBuffer.appendBuffer(chunk);
+  if (transport) {
+    await fetch('http://example.com/babyshark');
```

Comment:
Is this intended to be used?

Reply:
Do you mean the result?  The idea here is that the result comes in datagrams, not in the body of the response.